perception/map_based_prediction/include/path_generator.hpp

```diff
@@ -0,0 +1,104 @@
+// Copyright 2021 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef PATH_GENERATOR_HPP_
```

Comment:
MAP_BASED_PREDICTION__PATH_GENERATOR_

Reply:
done

Reply:
I can't write this way because the pre-commit doesn't allow this form. Could you tell me how to fix the macro this way without being forcibly modified by pre-commit?

Reply:
sorry. 
`MAP_BASED_PREDICTION__PATH_GENERATOR_HPP_` is right

Reply:
As I mentioned above, I cannot set the including guard like you proposed because pre-commiter does not allow this form.

Reply:
(summon @kenji-miyake )

Reply:
@kenji-miyake Can you give us some advice to fix this problem?

Reply:
I think you have to move this file to `map_based_prediction/include/map_based_prediction/path_generator.hpp`.

Reply:
done